### PR TITLE
Add bloom filter enabling info

### PR DESCRIPTION
**Pull request by QuentinBisson**
Closes https://github.com/giantswarm/roadmap/issues/3563#issuecomment-2252461858

We decided that bloom filters were a nice experiment for now but this is too volatile and has too many unknowns right now to be even remotely useful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +230,37 @@ Both subfields also need to have their `host` and `service` specified. If you de
...
+Giant Swarm experimented with bloom filters quite early one after the release of Loki 3.1.0 as can be seen [here](https://github.com/giantswarm/roadmap/issues/3563).
+
+You can quite easily enable blooms in your loki instance by setting the following configuration:
+
+```yaml
+loki:
+  loki:
+    structuredConfig:
+      bloom_compactor:
+        enabled: false
```
Comment (marieroque) [inline README.md:244]: how do we enable it ?
There is `bloom_compactor.enabled` and also `limits_config.bloom_compactor_enable_compaction`.

Comment (QuentinBisson) [reply README.md:244]: You need both. 
`bloom_compactor.enabled` enables the component
`limits_config.bloom_compactor_enable_compaction` enables the feature

Comment (marieroque) [reply README.md:244]: Ok, so I suggest to update the config example then.

Comment (marieroque) [inline README.md:244]:
```suggestion
        enabled: true
```

File: README.md [hunk 2]
```diff
@@ -230,6 +230,37 @@ Both subfields also need to have their `host` and `service` specified. If you de
...
+        retention:
+          enabled: true
+          max_lookback_days: 30
+      bloom_gateway:
+        enabled: false
```
Comment (marieroque) [inline README.md:249]:
```suggestion
        enabled: true
```

**Review by marieroque — APPROVED** [review]

**Pull request merged**